R/bayesfactor_models.R

```diff
@@ -178,7 +176,11 @@ bayesfactor_models.default <- function(..., denominator = 1, verbose = TRUE) {
   }
 
   # Get BF
-  mBIC <- .BIC_list(mods)
+  if (verbose) do.call(BIC, unname(mods))
```

Comment:
What's the purpose of this code?

Reply:
It produces a warning when the models were not fit with the same number of observations

``` r
m1 <- lm(mpg ~ hp, mtcars)
m2 <- lm(mpg ~ hp + gear, mtcars, subset = cyl > 4)

BIC(m1, m2)
#> Warning in BIC.default(m1, m2): models are not all fitted to the same number of observations
#>    df       BIC
#> m1  3 185.63584
#> m2  4  99.89051
```

<sup>Created on 2022-03-03 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

Reply:
You _could_ use `insight::ellipsis_info()` instead, a function @DominiqueMakowski wrote to evaluate the input of multiple objects.

Reply:
Looking at the internals of that function, it seems to be doing a lot for what I need here...

Perhaps if @DominiqueMakowski would export the `same_response` function?